File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.lua
n = io.read("*number")
    
for i=1,n,1 do
    x = io.read("*number")
    res = ""
    if(x==0) then
        print("NULL")
    else
        if(x%2==0) then
            res=res.."EVEN "
        else
            res=res.."ODD "
        end
        if(x>0) then
            res=res.."POSITIVE"
        else
            res=res.."NEGATIVE"
        end
        print(res)
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1074.rb
n = gets.to_i
    
for i in 1..n do
    x = gets.to_i
    res = ""
    if(x==0) then
        puts "NULL"
    else
        if(x%2==0) then
            res+="EVEN "
        else
            res+="ODD "
        end
        if(x>0) then
            res+="POSITIVE"
        else
            res+="NEGATIVE"
        end
        puts(res)
    end
end